image_definitions/data/Dockerfile

```diff
@@ -16,8 +16,11 @@ RUN echo "https://dl-cdn.alpinelinux.org/alpine/v3.17/community" >> /etc/apk/rep
 RUN apk update
 
 # Add latest version of python  and poetry
-RUN apk add --update python3 py3-pip gcc musl-dev linux-headers python3-dev
+RUN apk add --update python3 py3-pip gcc musl-dev linux-headers python3-dev jq
 RUN pip install pytest pylint pylint-exit pytest-azurepipelines pytest-cov poetry
 
 # Install the databricks CLI, precommit and tomli
-RUN pip install databricks-cli pre-commit tomli
\ No newline at end of file
+RUN pip install databricks-cli pre-commit tomli
```

Comment:
ideally add "azure-cli" here rather than separate line otherwise we create unnecessary docker layers, happy to approve now.

Reply:
Azure cli is already in the base image?

Reply:
Hi Jack, I can see its in base image but I think its getting removed from base image ,please refer this 
# Remove old python and pip versions
RUN  apk del py-pip \  
     && apk del  python3-dev py3-pip \
     && rm -rf ~/.cache/pip \ 
     && apk del  python3

Reply:
Sorry, juust catching up with PRs 

Yes I was just looking at this.

If we are upgrading Python herte then I think we should be consistent and upgrage Python in the base image.
I do not want to have to install `az` twice.